### ISSUE TITLE
Is this repository...anything?

### ISSUE THREAD

**Issue by mlissner** [freelawproject/tesseract-docker#1]
Should we just delete it or do we use it for something? @flooie, any thoughts?

**Comment by flooie** [comment]
Oh I think we use it.

**Comment by mlissner** [comment]
Huh. We should do some cleanup and make it more clear what it is and how to use it. I'm at a loss. :)

**Comment by flooie** [comment]
Yes, so to be clear - this repository holds the docker file for building tesseract v5. I found v5 which was not yet fully released to be better at generating results than v4.xxx.  So this repo is meant to really only hold the docker file for the tesseract version we use.

**Comment by mlissner** [comment]
OK, but we're not actually using it for anything?

**Comment by flooie** [comment]
Well...

**Comment by mlissner** [comment]
So anyway...should we delete it or update the readme? I'm happy to delete it. If it *does* have value, would you update the readme so we know what it is? 

Does this repo spark joy?

**Comment by mlissner** [comment]
Re-upping this issue, Bill. If we're not using this repo, let's kill it.

**Comment by flooie** [comment]
yes -we can move on from this repo

**Comment by flooie** [comment]
if memory serves at one point we needed a way to install tesseract-5 and get it into doctor.  But it was eventually solved in other ways.  so I think we can delete or archive this.
